src/utils/style.ts

```diff
@@ -11,3 +15,44 @@ export const opacityToClassName = (opacity: Opacity) => {
   }
   return className;
 };
+
+const createSpacingVariableFromKey = (key: Spacing) => {
```

Comment:
Cut out the process of generating a Style object that specifies the value of token through CSS Variables